LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 
+Copyright (c) 2025 
```

Comment:
This is not necessary. The year here is the year of "the first publication of the work". It does not need to be bumped constantly.